gpu_serving/object-detection-app/environments/kai4/globals.libsonnet

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Lets not check in your environment.
I'd suggest adding it to the .gitignore file so it doesn't get checked in.

Reply:
Done.
But it's still in app.yaml. Not sure if there is a good way to avoid that.